tests/debugger_test.py

```diff
@@ -29,6 +29,7 @@
 import jax.numpy as jnp
 import numpy as np
 
+config.update("jax_traceback_filtering", "off")
```

Comment:
After this test exits, this config setting might affect subsequent tests. Maybe handle this with setUp / tearDown module-level functions or class-level methods?